.github/workflows/deploy-docs.yaml

```diff
@@ -40,4 +40,3 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           latest: ${{ github.event_name != 'pull_request_target' && github.ref_name == github.event.repository.default_branch }}
-          mkdocs-requirements-txt: ./mkdocs-requirements.txt
```

Comment:
This pr cannot be merged because of this line.
After merging https://github.com/autowarefoundation/autoware-github-actions/pull/190, sync-files can be merged because mkdocs-requirements.txt becomes not necessary

Reply:
@hsgwa  
I issued the same pull request in the past. @kenji-miyake suggested me to add a function to customize project-dedicated mkdocs-requirements-txt.
https://github.com/autowarefoundation/autoware-github-actions/pull/163

I think that CARET project has to take care of this sync-files, so that we have to exclude or suppress this deletion.

Reply:
related: https://github.com/tier4/CARET_analyze/pull/175